src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -206,7 +206,8 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
         Future {
           try {
             val updatedWorkspace = applyOperationsToWorkspace(workspaceContext.workspace, operations)
-            RequestComplete(StatusCodes.OK, containerDAO.workspaceDAO.save(updatedWorkspace, txn).workspace)
+            containerDAO.workspaceDAO.save(updatedWorkspace, txn)
+            RequestComplete(StatusCodes.OK, containerDAO.workspaceDAO.loadContext(updatedWorkspace.toWorkspaceName, txn).head.workspace)
```

Comment:
you should not have to load the context here, save should return the right thing and if it doesn't, make it